Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -271,6 +277,10 @@ public async Task it_returns_diagnostics(Language language)
             {
                 Language.FSharp => "input.fsx (1,1)-(1,7) typecheck error The value or constructor 'aaaadd' is not defined.",
                 Language.CSharp => "(1,1): error CS0103: The name 'aaaadd' does not exist in the current context",
+                Language.PowerShell => @"At line:1 char:5
```

Comment:
This is a verbatim string, new lines will be captured from the OS where the code was prepared and not from the current machine. This could lead to flaky tests. if you are testing multilines maybe better us string join with the OS NewLine character or find other ways.

Reply:
This test discovered a bug in PowerShell - a hardcoded `\n` https://github.com/PowerShell/PowerShell/blob/ed1f6e3144a2f6fa78e69582870e29ae898e1e78/src/System.Management.Automation/engine/parser/Parser.cs#L8062

I'll be sending a PR to PowerShell to fix that but as a workaround, I'm replacing `\r\n` with `\n` which will work generically.

Reply:
PR to fix PowerShell: https://github.com/PowerShell/PowerShell/pull/11746